src/attestation/Attester.ts

```diff
@@ -37,11 +37,13 @@ export default class Attester implements IAttester {
    *
    * @param validityDuration The duration in days for which the public key will be valid.
    * @param maxAttributes The maximum number of attributes that can be signed with the generated private key.
+   * @param keyLength The key length of the new key pair. Note that this key will only support credentials and claimer with the same key length.
    * @returns A newly generated key pair.
    */
   public static async genKeyPair(
     validityDuration?: number,
-    maxAttributes = 70
+    maxAttributes = 70,
+    keyLength: 1024 | 2048 | 4096 = 1024
```

Comment:
If I want to generate a keyPair with > 1024 bits, I have to submit a validity duration and attributes. I would prefer to move keyLength to the top as it is the easiest parameter and maybe we should add a default validity duration.

Reply:
i will add an option parameter. You shouldn't be forced to add parameter you don't want to provide...